extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -51,7 +51,7 @@ amp-embed iframe {
   border-radius: 2px;
   color: #fff;
   font-size: 16px;
-  line-height: 16px;
+  line-height: 100%;
```

Comment:
line-height is 1 by default (the unit is em) which means it is the same as the computed/set font-size of the element, 1em = 100% ,